doc/guides/model-sync.md

```diff
@@ -7,7 +7,7 @@ hatchedKoa.modelSync()
 ```
 
 - This creates the table if it does not exist (and does nothing if it already exists)
-- Postgres: Namespaces (Postgres Schemas) are handled manually
+- Postgres: Namespaces (Postgres Schemas) are handled manually // 👀
```

Comment:
Our other supported database (sqllite) doesn't support namespace. It's listed this way since the namespaces behavior is postgres specific. I agree it looks a bit messy as is. 

If you have a suggestion on how to reformat this while still making it clear that these namespace changes are postgres-specific I'm happy to change it.

Reply:
I just spaced this out a bit.